UPGRADE.rst

```diff
@@ -75,6 +75,18 @@ for example:
      wget https://packages.matrix.org/debian/pool/main/m/matrix-synapse-py3/matrix-synapse-py3_1.3.0+stretch1_amd64.deb
      dpkg -i matrix-synapse-py3_1.3.0+stretch1_amd64.deb
 
+Upgrading to v1.20.0
```

Comment:
we're going to need to make sure that this and #8004 land in the same release, otherwise this is going to be confusing.